packages/miniflare/src/plugins/assets/index.ts

```diff
@@ -98,16 +87,64 @@ export async function getAssetsServices(
 					service: { name: storageServiceName },
 				},
 				{
-					name: "__STATIC_ASSETS_REVERSE_MAP",
+					name: "ASSETS_REVERSE_MAP",
 					json: assetsReverseMap,
 				},
 			],
 		},
 	};
-	return [storageService, namespaceService];
+
+	const assetsManifest = await buildAssetsManifest(options.assetsPath);
+	const assetService: Service = {
+		name: ASSETS_SERVICE_NAME,
+		worker: {
+			compatibilityDate: "2024-08-01",
+			modules: [
+				{
+					name: "asset-worker.mjs",
+					esModule: SCRIPT_ASSETS(),
+				},
+			],
+			bindings: [
+				{
+					name: "ASSETS_KV_NAMESPACE",
+					kvNamespace: { name: FAKE_KV_SERVICE_NAME },
+				},
+				{
+					name: "ASSETS_MANIFEST",
+					data: assetsManifest,
+				},
+			],
+		},
+	};
+
+	const routerService: Service = {
+		name: ROUTER_SERVICE_NAME,
+		worker: {
+			compatibilityDate: "2024-08-01",
```

Comment:
same here. Ideally we want AW and RW to be the source of truth for these things..as much as possible and makes sense ofc

Reply:
this is tracked in WC-2636